ambry-store/src/test/java/com.github.ambry.store/BlobStoreStatsTest.java

```diff
@@ -885,9 +887,18 @@ private long verifyAndGetContainerValidSize(BlobStoreStats blobStoreStats, long
       }
       assertEquals("Mismatch in number of containerIds in serviceId: " + serviceId, innerMap.size(),
           actualContainerValidSizeMap.get(serviceId).size());
+      actualContainerValidSizeMap.remove(serviceId);
+    }
+    for (Map.Entry<String, Map<String, Long>> actualContainerValidSizeEntry : actualContainerValidSizeMap.entrySet()) {
+      if (actualContainerValidSizeEntry.getValue().size() != 0) {
+        assertEquals("Additional values found in actual container valid data size map ", 1,
```

Comment:
I am not sure how it was working before. I can explain offline why this is required.